src/AppInstallerRepositoryCore/Rest/HttpClientHelper.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "cpprest/http_client.h"
+#include "cpprest/json.h"
+#include "HttpClientHelper.h"
+
+namespace AppInstaller::Repository::Rest
+{
+    HttpClientHelper::HttpClientHelper(const utility::string_t& url) : m_client(url), m_url(url) {}
+
+    pplx::task<web::http::http_response> HttpClientHelper::Post(const web::json::value& body)
+    {
+        AICLI_LOG(Repo, Verbose, << "Sending http POST request to: " << utility::conversions::to_utf8string(m_url));
+        web::http::http_request request{ web::http::methods::POST };
+        request.headers().set_content_type(web::http::details::mime_types::application_json);
+        request.set_body(body.serialize());
+        return MakeRequest(request);
+    }
+
+    web::json::value HttpClientHelper::HandlePost(const web::json::value& body)
+    {
+        web::json::value result;
+        web::http::http_response httpResponse;
+        HttpClientHelper::Post(body).then([&httpResponse](const web::http::http_response& response)
+            {
+                AICLI_LOG(Repo, Verbose, << "Response status: " << response.status_code());
+                httpResponse = std::move(response);

```

Comment:
>httpResponse = std::move(response); [](start = 16, length = 35)

Shouldn't move this as it is not an rvalue (&&).  I would suggest just capturing the status_code.